### DISCUSSION TITLE
Unusual symbol, abbreviation for thousand?

### DISCUSSION THREAD

**Discussion by afarlie** [psb1558/Junicode-font#234] [Q&A] [answered]
In a recent transcription I came across an abbreviation for thousand (based on context) I had not encountered before, 

In context the abbreviation occurs in a paragraph that seems to be some kind of declaration on the front of a 16th/17th century parish register.

The abbreviation is towards the end of   line 4  paragraph 1, of pp. 1 of  being pp. 1 of "A Transcript of the First Volume, 1538-1636, of the Parish Register of Chesham: in the County of Buckingham" by  John William Garrett Pegge, published in 1904

Image of scan page:-
  https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/A_transcript_of_the_first_volume%2C_1538-1636%2C_of_the_parish_register_of_Chesham_in_the_county_of_Buckingham.djvu/page23-1652px-A_transcript_of_the_first_volume%2C_1538-1636%2C_of_the_parish_register_of_Chesham_in_the_county_of_Buckingham.djvu.jpg

What is the abbreviation concerned, and is this symbol in Junicode?

**Comment by psb1558** [comment, marked as answer]
It's obvious that the whole assembly from the italic *a* to the superscript o has got to be an abbreviation for *millesimo*, but it makes no sense at all for an abbreviation for *millesimo* to begin with the letter *a*, and I have to wonder if the transcriber has messed up here. For comparison, look at the abbreviations for *millesimo* on p. 417 of Cappelli's *Dizionario*, where it would be easy to misinterpret the closed left side of the gothic *M* as an *a*:
<img width="273" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/ca87ba35-1125-45ee-beba-58e1230fc364">
So heaven knows what the transcriber is trying to represent with the scribble between *a* and *o*, which to my eye resembles at least vaguely the flourishes on the conventional M (presumably millesimo) signs encoded by MUFI at U+F2F2 and U+F2F3 (and found at those code points in Junicode).
<img width="181" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/a9943e44-5e43-45a1-8c51-9a826824c853">
I would have to see the MS before making any further guesses about what's going on here.

**Comment by afarlie** [comment]
Thanks, I suspect it's a mistake by the transcriber or printers based on context. ~~~~

**Comment by psb1558** [comment]
Or to be a little more charitable, the transcriber (or the printer) may just be doing his best to represent visually an assembly that has no counterpart in the font of metal type. So that *a* and part of the flourish would be representing the *m*:
<img width="250" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/3869bf56-6bd2-44ad-b303-f2c84c9280d1">
with the stylized *s* of the *millesimo* abbreviations represented by the rest of the flourish:
<img width="236" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/0a7886c5-8ba4-4633-8f9e-9b50222da3a4">
He would trust his readers to interpret the mess correctly because they all learned Latin in school at that time.

If you're trying to transcribe this, you could do worse than using U+F2F3 followed by ordmasculine U+00BA:
<img width="86" alt="image" src="https://github.com/psb1558/Junicode-font/assets/16215767/811d2e04-3ae6-46cc-85a8-c74973f298d7">

**Comment by afarlie** [comment]
This was my approach -https://en.wikisource.org/wiki/Page:A_transcript_of_the_first_volume,_1538-1636,_of_the_parish_register_of_Chesham_in_the_county_of_Buckingham.djvu/23
I did the best I could, and left an note on the page.

**Reply by psb1558** [reply]
That ought to do it.
